pallets/task/src/benchmarking.rs

```diff
@@ -224,6 +224,7 @@ benchmarks! {
 		let _ = PalletTask::<T>::create_task(RawOrigin::Signed(caller_create.clone()).into(), title, specification, budget, x.into());
 		let hash_task = PalletTask::<T>::tasks_owned(&caller_create)[0];
 		let _ = PalletTask::<T>::start_task(RawOrigin::Signed(caller_complete.clone()).into(), hash_task.clone());
+		let _ = PalletTask::<T>::complete_task(RawOrigin::Signed(caller_complete.clone()).into(), hash_task.clone());
```

Comment:
nice catch

Reply:
Actually , running test with `runtime-benchmarks` feature enable showed this failure.